src/proto/schemas/domains/Permissions.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+import "domains/Nodes.proto";
+import "domains/Identities.proto";
```

Comment:
How are we deciding whether a proto file should be capitals or lowercase? Originally they were capitals because they were `service Agent` and part of `package agentInterface`.

Can you check the grpc/protobuf spec, and explain which files should be capitals, which should be lowercase based on naming standards of services vs packages.

Reply:
See: https://developers.google.com/protocol-buffers/docs/style

Reply:
The style guide doesn't mention what should be done for file names. so far I've just had them as capital like `Vaults.proto` but I think we can decide on convention here.

Reply:
They say it should always be lower case:

> Files should be named lower_snake_case.proto

https://developers.google.com/protocol-buffers/docs/style#file_structure